### DISCUSSION TITLE
Fox Crime | (Slovenia)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#60] [Show and tell]
### Channel Name

Fox Crime

### Channel ID

FoxCrimeSerbia.rs

### Stream URL

http://194.163.179.246/slovenci/filmski/foxcrime.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
<img width="651" alt="image" src="https://user-images.githubusercontent.com/7253922/185268495-232fa966-1a59-4de1-b30b-92b5a5865120.png">
